backend/editor/entries.py

```diff
@@ -101,4 +186,40 @@ def update_nodes(label, entry, incomingData):
 
     params = dict(incomingData, id=entry)
     result = session.run(" ".join(query), params)
-    return result
\ No newline at end of file
+    return result
+
+def update_node_children(entry, incomingData):

```

Comment:
`incomingData` is not descriptive enough here. If I read correcly this is the list of children id, so you should rename it `children_ids` or `new_ids`.